### ISSUE TITLE
Jetpack portfolio - Testimonials support for Callout

### ISSUE THREAD

**Issue by fxbenard** [samikeijonen/mina-olen#7]

**Comment by samikeijonen** [comment]
Was this reminder to add support for Jetpack portfolio and testimonials?

Front page support can be tricky but other basic support will be easier to add.

**Comment by fxbenard** [comment]
Yes it is, if you use them, portfolios don't shows up on frontpage. Only the area title. After they work every else. See by yourself https://demos.hexagone.io/mina-olen/

**Comment by samikeijonen** [comment]
One reason for this is that every plugin author have different types of slugs for same post type. It would be cool if people were actually using content type standards. 

https://github.com/justintadlock/content-type-standards

Now we need to add if - else checks for different plugins if we want to support them in page templates. Archives always works and fallback for index.php.

**Comment by fxbenard** [comment]
Ok great I will not push users to use jetpack but Custom Content Portfolio even if @justintadlock didn't updated it for more than 2 years ;)
But you're right if you start using if - else checks you'll never ed up adding new rules

**Comment by samikeijonen** [comment]
I'm gonna check tomorrow is there an easy way to add Jetpack support for Portfolio and Testimonials.

**Comment by fxbenard** [comment]
Great, Jetpack might be someday kind of standard for portfolio and testimonials cpt

**Comment by samikeijonen** [comment]
Yeah but not for post type names. They use `jetpack` prefix in post type names. Not so good:)

**Comment by justintadlock** [comment]
I'm in the process of updating CCP to support the custom post type standards project.  We should reach out to portfolio plugin developers about supporting it or getting us all together and making an uber-portfolio plugin.

**Comment by samikeijonen** [comment]
Thanks @justintadlock for comment. Looking forward to that update.

**Comment by fxbenard** [comment]
great news, we'll definitely wait for this update. @samikeijonen no rush ;)
